UnitySDK/Assets/ML-Agents/Scripts/DecisionAutoRequester.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
```

Comment:
Do we have to add this monobehaviour to every Agent in all of our example environments?